scripts/package/templates/umd-msg.template

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable */
+;(function(root, factory) {
+  if (typeof define === 'function' && define.amd) { // AMD
+    define(<%= amd %>, factory);
+  } else if (typeof exports === 'object') { // Node.js
+    module.exports = factory();
+  } else { // Browser
+    Object.assign(root.<%= namespace %>, factory());
```

Comment:
This will not be compatible with IE11.  As discussed: it should be fine to use `.setLocale` here.